src/Servers/Connections.Abstractions/ref/Microsoft.AspNetCore.Connections.Abstractions.netstandard2.0.cs

```diff
@@ -30,12 +30,16 @@ public static partial class ConnectionBuilderExtensions
     public abstract partial class ConnectionContext
```

Comment:
File an issue to make IDisposable.

Reply:
https://github.com/aspnet/AspNetCore/issues/10607